### PR TITLE
Use Ubuntu 22.04 (jammy) base image

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
Ubuntu 22.04 (jammy) will be LTS in 2022 April
https://ubuntu.com/about/release-cycle
This is done to mitigate issues in Ubuntu 20.04

### Issues

Related to https://github.com/wso2/product-microgateway/issues/2412

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments

Not Tested
Ubuntu AMD64 integration tests - https://github.com/renuka-fernando/product-microgateway/actions/runs/1507353083

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: enforcer-parent/enforcer/src/main/resources/Dockerfile [hunk 1]
```diff
@@ -15,9 +15,9 @@
 # -----------------------------------------------------------------------
 
 FROM adoptopenjdk/openjdk11:jre-11.0.13_8-alpine
-RUN apk update && apk upgrade --no-cache
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>"
```
Comment (pubudu538) [inline enforcer-parent/enforcer/src/main/resources/Dockerfile:18]: Shall we use wso2.com as the Maintainers.

Comment (pubudu538) [reply enforcer-parent/enforcer/src/main/resources/Dockerfile:18]: Let's change this in all the places

Comment (renuka-fernando) [reply enforcer-parent/enforcer/src/main/resources/Dockerfile]: Updated with https://github.com/wso2/product-microgateway/pull/2497/commits/e46f1559aaad908557b6ce0d62ea4c564beb2173

**Review by praminda — APPROVED** [review]
22.04 is yet to be released using this image might have other concerns. But due to issues identified in 20.04 LTS, I'm ok to offer the ubuntu 22.04 images for try outs.

**Comment by renuka-fernando** [comment]
Closing this, using ubuntu 20LTS as a tryout image
Related Dockerfile improvements done in this pr is converted in https://github.com/wso2/product-microgateway/pull/2504

**Pull request closed without merging**